onadata/apps/api/migrations/0009_oauthstoragemodel_provider.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.23 on 2021-07-28 13:49
+
```

Comment:
Lets combine the migrations in this PR into one migration

Reply:
done